core/src/mindustry/entities/comp/MinerComp.java

```diff
@@ -123,4 +123,36 @@ public void update(){
             }
         }
     }
+
+    public void drawMining(){
```

Comment:
This should be in UnitType, that's where the rest of the mining code is right now.